schemas/responseOptions

```diff
@@ -0,0 +1,206 @@
+                }
+            ],
+            "schema:sameAs": {
+                "@id": "schema:image"
+            }
+        },
+        {
+            "@id": "reproterms:multipleChoice",
+            "@type": "rdf:Property",
+            "rdfs:comment": "Indicates (by bool) if response for the Field item is a multiple choice type.",
+            "rdfs:label": "multipleChoice",
+            "schema:domainIncludes": {
+                "@id": "reproterms:responseOptions"
+            },
+            "schema:rangeIncludes": {
+                "@id": "schema:Boolean"
+            },
+            "schema:sameAs": {
+                "@id": "reproterms:multipleChoice"
+            }
+        },
+        {
+            "@id": "reproterms:requiredValue",
+            "@type": "rdf:Property",
+            "rdfs:comment": "Indicates (by bool) if the response for the Field item is mandatory or not.",
+            "rdfs:label": "requiredValue",
+            "schema:domainIncludes": {
+                "@id": "reproterms:responseOptions"
+            },
+            "schema:rangeIncludes": {
+                "@id": "schema:Boolean"
+            },
+            "schema:sameAs": {
+                "@id": "reproterms:requiredValue"
+            }
+        },
+        {
+            "@id": "reproterms:datumType",
+            "@type": "rdf:Property",
+            "rdfs:comment": "Indicates what type of datum the response is (e.g. range,count,scalar etc.) for the Field item.",
+            "rdfs:label": "datumType",
+            "schema:domainIncludes": {
+                "@id": "reproterms:responseOptions"
+            },
+            "schema:rangeIncludes": {
+                "@id": "schema:text"
+            },
+            "schema:sameAs": {
+                "@id": "reproterms:datumType"
+            }
+        },
+        {
+            "@id": "schema:unitCode",
+            "@type": "rdf:Property",
+            "rdfs:comment": "The unit of measurement given using the UN/CEFACT Common Code (3 characters) or a URL. Other codes than the UN/CEFACT Common Code may be used with a prefix followed by a colon."
+            "rdfs:label": "unitCode",
+            "schema:domainIncludes": {
+                "@id": "reproterms:responseOptions"
+            },
+            "http://schema.org/rangeIncludes": [
+                {
+                    "@id": "schema:URL"
+                },
+                {
+                    "@id": "schema:Text"
+                }
+            ],
+            "schema:sameAs": {
+                "@id": "schema:unitCode"
+            }
+        },
+        {
+            "@id": "reproschema:choices",
+            "@type": "rdf:Property",
+            "rdfs:comment": "An array to list the available options in responseOptions of the Field item.",
+            "rdfs:label": "choices",
+            "schema:domainIncludes": {
+                "@id": "reproschema:responseOptions"
+            },
+            "schema:rangeIncludes": {
```

Comment:
@satra what should be the `rangeIncludes` for `choices` ?

Reply:
range is always about the value of the property - in this case, it should be an object that describes the properties of those choices + skip + don't know.